compose/const.py

```diff
@@ -15,6 +15,8 @@
 LABEL_VERSION = 'com.docker.compose.version'
 LABEL_CONFIG_HASH = 'com.docker.compose.config-hash'
 
+PATH_SEPARATOR = ';' if IS_WINDOWS_PLATFORM else ':'
```

Comment:
I think we can use [`os.pathsep`](https://docs.python.org/2/library/os.html#os.pathsep) here?

> The character conventionally used by the operating system to separate search path components (as in `PATH`), such as `':'` for POSIX or `';'` for Windows. Also available via `os.path`.